pkg/resolver/searchSubscription.go

```diff
@@ -0,0 +1,46 @@
+// Copyright Contributors to the Open Cluster Management project
+package resolver
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/stolostron/search-v2-api/graph/model"
+)
+
+func SearchSubscription(ctx context.Context, input []*model.SearchInput) (<-chan []*SearchResult, error) {
+	ch := make(chan []*SearchResult)
+
+	// You can (and probably should) handle your channels in a central place outside of `schema.resolvers.go`.
+	// For this example we'll simply use a Goroutine with a simple loop.
+	go func() {
+		// Handle deregistration of the channel here. Note the `defer`
+		defer close(ch)
+
+		for {
+			// Send the search results every 10 seconds.
+			time.Sleep(10 * time.Second)
+			fmt.Println("SearchSubscription new poll interval")
+			
+			// TODO 2nd return item is error... needs to be handled
+			searchResult, _ := Search(ctx, input)
+
+			// The subscription may have got closed due to the client disconnecting.
+			// Hence we do send in a select block with a check for context cancellation.
+			// This avoids goroutine getting blocked forever or panicking,
+			select {
+			case <-ctx.Done(): // This runs when context gets cancelled. Subscription closes.
+				fmt.Println("Subscription Closed")
```

Comment:
We'll just want to convert this to a log instead of Println before merge

Reply:
As well as probably err log the above unhandled err. Otherwise from testing and what I can see in the graphql subscription docs looks good.